examples/with-iron-session/pages/api/user.js

```diff
@@ -0,0 +1,23 @@
+import withSession from '../../lib/session'
+
+export default withSession(async (req, res) => {
+  const user = req.session.get('user')
+  if (user !== undefined) {
+    try {
+      // Here you could not just read from the session but also do a database request or another API call
+      // to get a more complete user
+      // We do not call GitHub in /api/user because of GitHub API rate limits
```

Comment:
How is the GitHub API limits an issue here?

Reply:
The GitHub API limits when not authenticated is too low if you're in development with devtools or multiple windows, there are a LOT of requests being done to /api/user (everytime you focus/in/out). That's why

I removed the comment now